index.js

```diff
@@ -1,99 +1,109 @@
 'use strict';
 
-var replaceExt    = require('replace-ext');
-var through       = require('through2');
-var yaml          = require('js-yaml');
-var xtend         = require('xtend');
+var yaml = require('js-yaml');
```

Comment:
Maybe using const/let instead of var, where possible?

Reply:
Yes, absolutely. I only stuck to `var` because the code base only used `var` before refactoring.